R/constants-aglu-L2.R

```diff
@@ -0,0 +1,201 @@
+# Column names of tables that are read in to the model interface
+# These correspond to the headers of the same name in the headers/ModelInterface_headers.txt file
+# These are for generic categories; module-specific level2 data names are found in data header files of each module
+
+R <- "region"
+Y <- "year"
+input <- "minicam.energy.input"
```

Comment:
Sorry I didn't get a chance to work on this tonight. I just checked out the branch and I'll see what I can do this weekend.